cmd/integration_test/build/script/create_infra_config.sh

```diff
@@ -17,11 +17,22 @@ set -e
 set -x
 set -o pipefail
 
+REPO_ROOT=$(git rev-parse --show-toplevel)
+source $REPO_ROOT/test/e2e/E2E_AMI_FILTER_VARS
+
+INTEGRATION_TEST_AMI_ID=$(aws ec2 describe-images \
```

Comment:
nit: could check the content of this var before passing it to the config file? that way we can error out and make the error explicit if for whatever reason we can't find a valid AMI

Reply:
Added a check!